build.sh

```diff
@@ -142,6 +142,10 @@ apt-get -q -y --purge install language-pack-gnome-en
 apt-get -q -y --purge install ibus ibus-clutter ibus-gtk ibus-gtk3 ibus-qt4
 apt-get -q -y --purge install ibus-unikey ibus-anthy ibus-pinyin ibus-m17n
 apt-get -q -y --purge install im-switch
+
+#Debuilding the metapackages
+./script.sh
```

Comment:
the script should change the package before it is copied to chroot not inside the chroot